jmix-core/core/src/main/java/io/jmix/core/impl/repository/query/JmixCustomLoadQuery.java

```diff
@@ -85,15 +89,19 @@ public JmixCustomLoadQuery(DataManager dataManager, Metadata jmixMetadata, Metho
     @Override
     public Object execute(Object[] parameters) {
         UnconstrainedDataManager dataManager = getDataManager();
-
+        //todo taimanov: rework through LoadContext
```

Comment:
Why this todo is here? Should the code below be reworked?

Reply:
It is just refactoring do not affecting something. I will remove todo and make it during further work.